test/opt/fix_storage_class_test.cpp

```diff
@@ -916,6 +916,43 @@ TEST_F(FixStorageClassTest, SupportsU64Index) {
   SinglePassRunAndMatch<FixStorageClass>(text, false);
 }
 
+TEST_F(FixStorageClassTest, CorrectlyProcessAccessChainOnCoopMatrix) {
+  const std::string text = R"(OpCapability CooperativeMatrixKHR
```

Comment:
Is it expected that this test has no 'CHECK'?

Reply:
That is correct. This is not a Match test, which uses effcee. This test makes sure that the spir-v input is the exact same as the output.